src/odin/maneuversbuilder.cc

```diff
@@ -3008,7 +3028,10 @@ void ManeuversBuilder::MatchGuidanceViewJunctions(Maneuver& maneuver,
             (base_prefix == overlay_tokens.at(0))) {
           DirectionsLeg_GuidanceView guidance_view;
           guidance_view.set_data_id(std::to_string(trip_path_->osm_changeset()));
-          guidance_view.set_type("jct"); // TODO implement for real in the future based on sign type
+          guidance_view.set_type(
+              guidanceview_type_string
```

Comment:
just assign the enum here and then translate to string in `route_serializer_osrm.cc`